src/Tribe/Field.php

```diff
@@ -349,7 +349,7 @@ public function do_screen_reader_label() {
 		 */
 		public function do_field_value() {
 			$return = '';
-			if ( $this->value ) {
+			if ( '' != $this->value ) {
 				$return = ' value="' . $this->value . '"';
 			}
```

Comment:
I couldn't *suggest* this, but it improves both the logic and comments -- consider if you disagree with this more explicit checking:

```
		/**
		 * Returns the field's value.
		 *
		 * @return string The field value.
		 */
		public function do_field_value() {
			$return = '';
			if ( 
				// License type fields shouldn't allow displaying empty values.
				! empty( $this->value )
				// Allow text fields to display '0' value, such as what to display for Free ticket.
				|| (
					'text' === $this->type
					&& '' !== $this->value
				)
			) {
```